packages/remix-server-runtime/__tests__/data-test.ts

```diff
@@ -76,6 +76,45 @@ describe("loaders", () => {
     expect(await res.headers.get("X-Remix-Catch")).toBeTruthy();
   });
 
+  it("doesn't set header for thrown 304 responses", async () => {
+    let loader = async ({ request }) => {
+      throw new Response("null", {
```

Comment:
```suggestion
      throw new Response(null, {
```

Reply:
I did this to be consistent with the existing test immediately above. Should I change both?

Reply:
I think that would be a good idea